app.py

```diff
@@ -3,24 +3,55 @@
 from flask import Flask
 from flask_restplus import Api, Resource
 from flask_sqlalchemy import SQLAlchemy
+from datetime import datetime
 
 app = Flask(__name__)
-app.config["SQALCHEMY_DATABASE_URL"] = os.getenv("APP_SQL_URL")
+app.config["SQLALCHEMY_DATABASE_URI"] = os.getenv("APP_SQL_URL")
+app.config['SQLALCHEMY_TRACK_MODIFICATIONS'] = False
 db = SQLAlchemy(app)
 api = Api(app, title="Review Requester")
 
-
 class Reviewer(db.Model):
```

Comment:
We need some way to track a many-to-many relationship between reviewers and the languages they're willing/able to review in. I can walk through [this doc](https://flask-sqlalchemy.palletsprojects.com/en/2.x/models/#many-to-many-relationships) with you if it will help.

Reply:
added lookup table